dato.config.js

```diff
@@ -68,6 +68,11 @@ function transformItem(item) {
     $('img').remove()
     item.body = $('body').html()
   }
+  else if (item.type === 'video') {
+    if (item.video && item.title) {
+      item.video.title = item.title
```

Comment:
I wouldn't put the `title` on the `item.video`. The `title` is the title of the `item`. The video contains all things related to the _external_ video source.